Source/AwsCommonRuntimeKit/event-stream/EventStreamHeader.swift

```diff
@@ -5,7 +5,7 @@ import AwsCEventStreams
 import Foundation
 
 public struct EventStreamHeader {
-    /// Header name length can not be greater than UInt8.max
+    /// Header name length can not be greater than Int8.max
```

Comment:
let's provide proper constants

```suggestion
    /// Header name length can not be greater than EventStreamHeader.maxNameLength
```

could just be a static?
`public static let maxNameLength = AWS_EVENT_STREAM_HEADER_NAME_LEN_MAX`